### DISCUSSION TITLE
Nuxt generate run sentry-webpack-plugin with incorrect order

### DISCUSSION THREAD

**Discussion by Fnxxxxo** [nuxt/nuxt#9354] [Questions (Nuxt 2)]
### Versions

- nuxt: 2.14.7 ~ 2.15.6
- node: 14.15.5

### Reproduction

Repo: https://github.com/Fnxxxxo/nuxt-sentry-webpack-plugin-issue
Steps:
1. `yarn install`
2. Add your sentry project configurations in config.env
3. `yarn generate`

<details open>
<summary>Additional Details</summary>
<br>
<pre markdown="1">
import SentryCliPlugin from '@sentry/webpack-plugin'

export default {
  // Disable server-side rendering: https://go.nuxtjs.dev/ssr-mode
  ssr: false,

  // Target: https://go.nuxtjs.dev/config-target
  target: 'static',

  srcDir: 'src',

  // Global page headers: https://go.nuxtjs.dev/config-head
  head: {
    title: 'demo',
    htmlAttrs: {
      lang: 'en'
    },
    meta: [
      { charset: 'utf-8' },
      { name: 'viewport', content: 'width=device-width, initial-scale=1' },
      { hid: 'description', name: 'description', content: '' }
    ],
    link: [
      { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }
    ]
  },

  // Build Configuration: https://go.nuxtjs.dev/config-build
  build: {
    extend(config, context) {
      if (config.resolve) {
        config.resolve.symlinks = false
      }
      if (!context.isDev) {
        config.devtool = 'source-map'
      }
      if (config.plugins) {
        config.plugins.push(new SentryCliPlugin({
          url: `http://${process.env.SENTRY_BASE}`,
          authToken: process.env.SENTRY_AUTH_KEY,
          include: 'dist',
          ignoreFile: '.gitignore',
          ignore: ['node_modules', 'webpack.config.js'],
          configFile: 'sentry.properties',
          org: 'sentry',
          project: process.env.SENTRY_PROJECT,
          debug: true
        }))
      }
      return config
    }
  }
}
</pre>
</details>

### Steps to reproduce
<p>When the fisrt execution, Nuxt complains <code>error: dist: IO error for operation on dist</code> but with dist directory generated and sentry-webpack-plugin fails to publish.</p>
<p>If I delete <code>node_modules/.cache</code> and generate agian. The process run successful but use the previous generated <code>dist</code> and map files.</p>
<p>If I delete both <code>.cache</code> and <code>dist</code>, I can reproduce the error.</p>

### What is Expected?
Generate `dist` then run sentry-webpack-plugin to publish it.

### What is actually happening?
Run sentry-webpack-plugin first and publish old `dist`

**Comment by danielroe** [comment]
The issue is how you're configuring `@sentry/webpack-plugin`. You're configuring at at the build stage (before Nuxt generates the `dist` folder) yet are asking it to look in the `dist` folder. More importantly, according to their docs, `include` is meant for _sources_, not built files. [Their example](https://github.com/getsentry/sentry-webpack-plugin#usage) has`include: '.'`.

**Reply by Fnxxxxo** [reply]
@danielroe I think the `include: '.'` means the scope of searching for maps so I just set it to `dist`. I found the community module `@nuxtjs/sentry` solves it.
